content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -80,6 +81,10 @@ Possible options:
 | `false`          | `admission.datadoghq.com/enabled=true`  | Yes       |
 | `false`          | `admission.datadoghq.com/enabled=false` | No        |
 
+## Configure the Tracer from a Different Registry
+You may configure the tracer from a different registry using the `admission_controller.auto_instrumentation.container_registry` config in the cluster agent or the `DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_CONTAINER_REGISTRY` environment variable.
```

Comment:
```suggestion
You may pyll the APM library from a different registry using the `DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_CONTAINER_REGISTRY` environment variable in the Datadog Cluster Agent.
```

In containerized environments environment variables are recommended to configure the agent, as it's easier to add them instead of changing and mouting a datadog.yaml file.

Reply:
+1, but "pull" not "pyll" (unless pyll is some python thing I was previously unaware of)

Reply:
Yes sorry that's a typo 🤦 it's pull.